.github/workflows/_test_jax.yaml

```diff
@@ -8,11 +8,20 @@ on:
         description: 'JAX image built by NVIDIA/JAX-Toolbox'
         required: true
         default: 'ghcr.io/nvidia/jax:latest'
+    outputs:
+      UNIT_TEST_ARTIFACT_NAME:
+        description: 'Name of the unit test artifact for downstream workflows'
+        value: ${{ jobs.unit-tests.outputs.UNIT_TEST_ARTIFACT_NAME }}
+
+env:
+  UNIT_TEST_ARTIFACT_NAME: unit-test-logs
 
 jobs:
 
   unit-test:
     runs-on: [self-hosted, compute, V100]
+    outputs:
+      UNIT_TEST_ARTIFACT_NAME: ${{ env.UNIT_TEST_ARTIFACT_NAME }}
```

Comment:
`ARTIFACT_NAME` should suffice since the workflow/jobs already has 'test' in their names.

For example, if we use it in a later job, it will be
```
needs.unit-tests.outputs.ARTIFACT_NAME
```